routes/index.js

```diff
@@ -14,10 +15,17 @@ router.get('/download/:email/:uniqueId', function (req, res, next) {
 });
 
 router.get("/github", function (req, res, next) {
-  req.session.uniqueId = req.query.uniqueId;
-  req.session.email = req.query.email
-  req.session.gitURL = req.query.gitURL
-  next()
+  if (validation.isGitSSH(req.query.gitURL) || validation.isGitHTTPS(req.query.gitURL)) {
```

Comment:
@sch00lb0y No need for that. we can't use ssh with webUI. just remove validation for SSH url!